payment-request/payment-request-abort-method.https.html

```diff
@@ -44,24 +44,22 @@
 
 promise_test(async t => {
   const request = new PaymentRequest(defaultMethods, defaultDetails);
-  const promises = new Set([request.abort(), request.abort(), request.abort()]);
+  const promises = new Set([
+    request.abort(),
+    request.abort(),
+    request.abort(),
+  ]);
   assert_equals(promises.size, 3, "Must have three unique objects");
 }, "Calling abort() multiple times is always a new object.");
 
 promise_test(async t => {
   const request = new PaymentRequest(defaultMethods, defaultDetails);
-  const [abortPromise, acceptPromise] = await test_driver.bless(
-    "show payment request",
-    () => {
-      const acceptPromise = request.show()
-      acceptPromise.catch(() => {}); // no-op, just to silence unhandled rejection in devtools.
-      const abortPromise = request.abort();
-      return [abortPromise, acceptPromise];
-    });
-
-  await abortPromise;
+  await test_driver.bless("show payment request");
```

Comment:
I'm not an expert on this, but thought `test_driver.bless()` takes a second argument as the action to be done with the user activation. If we're taking the `show()` and `abort()` calls out of `bless()`, are we relying on the user activation to be alive within some time window? I guess if it works I'm OK with it.

Reply:
Indeed, the second arg is optional. The "bless()" just resolves after the click and then time-based "transient activation" kicks in, which is time based or until someone "consumes" it. Makes things much nicer, as you don't need the inner function.